### PR TITLE
Remove cert caching

### PR DESCRIPTION
**Pull request by caseylitton**
@caesar2164 @stvstnfrd PR to remove cert caching

https://trello.com/c/Bc03HtUR/1011-remove-cert-caching-when-creating-certs-with-same-courseid-distiction-the-previous-cert-template-it-used

### CODE REVIEW

**Comment by caseylitton** [comment]
[Remove cert caching. When creating certs with same course_id/distiction the previous cert template it used.](https://trello.com/c/Bc03HtUR/1011-remove-cert-caching-when-creating-certs-with-same-courseid-distiction-the-previous-cert-template-it-used)

File: certificate_agent.py [hunk 1]
```diff
@@ -57,7 +57,6 @@ def main():
                                 settings.QUEUE_AUTH_USER,
                                 settings.QUEUE_AUTH_PASS,
                                 settings.QUEUE_USER, settings.QUEUE_PASS)
-    cert = None
     while True:
```
Comment (caesar2164) [inline certificate_agent.py:60]: Do we need this while loop?

**Review by caesar2164 — APPROVED** [review]
This looks good to me. 👍 

I added a comment, but it may not be pertinent to this PR.

File: certificate_agent.py (hunk 1 above)
Comment (stvstnfrd) [reply certificate_agent.py:60]: Yes.

**Review by stvstnfrd — APPROVED** [review]
+1

**Pull request merged**